realm/realm-library/src/androidTest/java/io/realm/internal/CollectionTests.java

```diff
@@ -398,13 +399,16 @@ public void onChange(Collection collection1) {
                     case 1:
                         assertEquals(collection1.size(), 5);
                         sharedRealm.close();
-                        looperThread.testComplete();
+                        break;
+                    default:
+                        fail();
```

Comment:
What will happen when reached to here after `looperThread.testComplete();` is called?
But yes, it's difficult to check that the listener is **not called after** `looperThread.testComplete();` is executed.
It's just a question from my curiosity.